make.js

```diff
@@ -86,6 +86,20 @@ target.test = function() {
     run('mocha test/tests');
 }
 
+target.testNoBuild = function() {
```

Comment:
Can we just make test be no build instead of having two?

Reply:
I could go either way on this. I think its nice to build before testing in general because it removes that step. I updated for better modularity though so its less of a big deal. If you feel strongly I'm also ok getting rid of this

Reply:
Yeah let's get rid of it and keep them separate.